reporter/views.py

```diff
@@ -100,7 +100,20 @@ def roads():
         abort(500)
     else:
         try:
-            file_handle = get_osm_file(bbox, coordinates)
+            url_path = (
+                'http://overpass-api.de/api/interpreter?data='
+                '('
+                'node["highway"]'
+                '({SW_lat},{SW_lng},{NE_lat},{NE_lng});'
+                'way["highway"]'
+                '({SW_lat},{SW_lng},{NE_lat},{NE_lng});'
+                'relation["highway"]'
+                '({SW_lat},{SW_lng},{NE_lat},{NE_lng});'
+                ');'
+                '(._;>;);'
```

Comment:
</sarcasm>One of the reasons I love overpass <sarcasm> :-) The syntax is a bit arcane :-P Great fix though!